metadata.yaml

```diff
@@ -48,6 +48,8 @@ provides:
   cni:
     interface: kubernetes-cni
     scope: container
+  website:
```

Comment:
@radraw i know there are a lot of charms with rev proxy capabilities that call this relation `website`, but this feels too generic to me. Would you consider calling it `ingress-proxy` instead?  That way it's more clear as to what we expect to happen when apache/haproxy are related to k8s-worker.